docs/content/testing/testing-guidelines.md

```diff
@@ -0,0 +1,252 @@
+**Warning:** The [pre-alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release is for evaluation purposes only, with limited documentation - see the guidance on [Recommended use](../../../README.md#recommended-use).
+
+-----
+
+# Testing guidelines
```

Comment:
Maybe "Test guidelines" because "Testing guidelines" reads like you are testing the guidelines.